core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.html

```diff
@@ -102,9 +102,14 @@ <h4 class="modal-title">Execution results of Workflow {{workflowName}}</h4>
               nzType="star"
               class="bookmark-icon"
               [nzTheme]="(row.bookmarked || currentlyHoveredExecution === row) ? 'fill' : 'outline'"
+              (mouseenter)="currentlyHoveredExecution = row"
```

Comment:
don't introduce this, it is removed in latest master

Reply:
done